lib/smart_answer/flow_registry.rb

```diff
@@ -53,12 +53,7 @@ def available?(name)
 
     def build_flow(name)
       class_prefix = name.tr("-", "_").camelize
-      if Rails.env.development?
-        load @load_path.join("#{name}.rb")
-      else
-        require @load_path.join(name)
-      end
```

Comment:
Will this still support automatic reloading of flows during development?

Reply:
Yup, they just don't need custom code for it as Zeitwerk takes care of it as part of Rails class reloading

It also fixes a problem that [any classes that use the `append` method](https://github.com/alphagov/smart-answers/blob/e36f68bedac1e35007f6e07dbbb7b2da95494be7/lib/smart_answer_flows/maternity-paternity-calculator.rb#L43) stop working when automatic reloading occurs and you get an error such as: `uninitialized constant SmartAnswer::MaternityPaternityCalculatorFlow::AdoptionCalculatorFlow`